avm/res/network/azure-firewall/version.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "0.2",
+    "version": "0.2.1",
```

Comment:
Hey @hundredacres, please note that patch/bug level updates are automatically applied by the CI. In other wrods, please undo, or update the minor ;)

Reply:
@AlexanderSehr  I revved the minor.